core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -76,6 +76,19 @@ export class LoggedInUser extends BaseUser {
       throw new Error(`Exploration with title ${title} is not present.`);
     }
   }
+
+  async expectTopicLinkReturns404(topicUrlFragment: string): Promise<void> {
+    await this.goto(`http://localhost:8181/learn/staging/${topicUrlFragment}`);
+    await this.page.waitForTimeout(1000);
+    const isError404Present = await this.isTextPresentOnPage('Error 400');
```

Comment:
The name of this variable mentions error 404 but you are trying to find 'Error 400'? Is it just a typo? Please make sure both are consistent.

Reply:
updated.